src/features/classification/security-controls/__tests__/SecurityControls.test.js

```diff
@@ -45,6 +45,13 @@ describe('features/classification/security-controls/SecurityControls', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should render SecurityControls with Integration label when using SHORT controlsFormat and shouldDisplayAppsAsIntegrations is true', () => {
+        wrapper.setProps({ controlsFormat: SHORT, shouldDisplayAppsAsIntegrations: true });
+        expect(wrapper.find('SecurityControlsItem').prop('message').id).toBe(
+            messages.shortSharingDownloadIntegration.id,
```

Comment:
we should be asserting against the hardcoded string, not the id or the variable, if the message changes, none of the new tests will fail if someone else accidentally changes it.  this goes for all the tests you are asserting below checking for the correct message to appear. 

also this component small enough to convert the test file to RTL where you could see the literal string in the dom so that its not mocked in the rest of the tests.